libs/market-depth/src/lib/orderbook-manager.tsx

```diff
@@ -129,8 +106,7 @@ export const OrderbookManager = ({ marketId }: OrderbookManagerProps) => {
 
   const marketDataUpdate = useCallback(
     ({ data }: { data: MarketData | null }) => {
-      marketDataRef.current = data;
-      updateOrderbookData.current();
+      setMarkPrice(data?.markPrice || '');
```

Comment:
Why not just let the internal `update` function handle the update. Then we can avoid storing an extra value in state and handling the update

I see no issue with just using markPrice like this
```
const {
    data: marketData,
    error: marketDataError,
    loading: marketDataLoading,
  } = useDataProvider({
    dataProvider: marketDataProvider,,
    variables,
  });
  
  // just use the value
  marketData.markPrice 
```

Reply:
done